app/assets/main/react_components/dashboard/climate_actions/carousel/CarouselList.jsx

```diff
@@ -34,21 +33,25 @@ const CarouselList = ({ user, updateLocalAccepted, categories }) => {
 
   const actions = isMobile ? sortForMobileClimateActions : climateActions;
 
+  const showNoOfActions = isMobile
```

Comment:
I don't understand this variable name. Maybe remove noOfActionsShown is better if the result is a number?